exporter/stats/stackdriver/stackdriver.go

```diff
@@ -48,8 +48,8 @@ type Exporter struct {
 	bundler *bundler.Bundler
 	o       Options
 
-	measuresMu sync.Mutex
-	measures   map[stats.Measure]struct{} // measures already created remotely
+	viewMu       sync.Mutex
```

Comment:
createdViewsMu

Always name the mutex after the field it is protecting.

Reply:
Done.